libass/ass_func_template.h

```diff
@@ -117,8 +117,8 @@ const BitmapEngine DECORATE(bitmap_engine) = {
     .mul_bitmaps = ass_mul_bitmaps_c,
 #endif
 
-#ifdef __x86_64__
-    .be_blur = DECORATE(be_blur),
+#if defined(__x86_64__) && (ENGINE_FLAGS & ASS_CPU_FLAG_X86_SSE2)
+    .be_blur = DECORATE_SUFFIX(be_blur, sse2),
```

Comment:
I think that simple `ass_be_blur_sse2` is better and `DECORATE_SUFFIX` is unnecessary. But if you'll choose to keep it then `ass_be_blur_c` below should be replaced accordingly.